assets/helpers/internationalisation/country.js

```diff
@@ -89,11 +145,70 @@ function fromString(s: string): ?IsoCountry {
     case 'uk': return 'GB';
     case 'us': return 'US';
     case 'au': return 'AU';
+    case 'ad': return 'AD';
+    case 'al': return 'AL';
+    case 'at': return 'AT';
+    case 'ba': return 'BA';
+    case 'be': return 'BE';
+    case 'bg': return 'BG';
+    case 'bl': return 'BL';
+    case 'ch': return 'CH';
+    case 'cy': return 'CY';
+    case 'cz': return 'CZ';
+    case 'de': return 'DE';
+    case 'dk': return 'DK';
+    case 'ee': return 'EE';
+    case 'es': return 'ES';
+    case 'fi': return 'FI';
+    case 'fo': return 'FO';
+    case 'fr': return 'FR';
+    case 'gf': return 'GF';
+    case 'gl': return 'GL';
+    case 'gp': return 'GP';
+    case 'gr': return 'GR';
+    case 'hr': return 'HR';
+    case 'hu': return 'HU';
+    case 'ie': return 'IE';
+    case 'it': return 'IT';
+    case 'li': return 'LI';
+    case 'lt': return 'LT';
+    case 'lu': return 'LU';
+    case 'lv': return 'LV';
+    case 'mc': return 'MC';
+    case 'me': return 'ME';
+    case 'mf': return 'MF';
+    case 'is': return 'IS';
+    case 'mq': return 'MQ';
+    case 'mt': return 'MT';
+    case 'nl': return 'NL';
+    case 'no': return 'NO';
+    case 'pf': return 'PF';
+    case 'pl': return 'PL';
+    case 'pm': return 'PM';
+    case 'pt': return 'PT';
+    case 're': return 'RE';
+    case 'ro': return 'RO';
+    case 'rs': return 'RS';
+    case 'se': return 'SE';
+    case 'si': return 'SI';
+    case 'sj': return 'SJ';
+    case 'sk': return 'SK';
+    case 'sm': return 'SM';
+    case 'tf': return 'TF';
+    case 'tr': return 'TR';
+    case 'wf': return 'WF';
+    case 'yt': return 'YT';
+    case 'va': return 'VA';
+    case 'ax': return 'AX';
```

Comment:
From the performance point of view, it depends on the platform which is running the code. Sometimes is O(1) and sometimes O(n). Since the number of countries it is not massive (the worst case is ok), I prefer to keep it as a switch statement for the time being.

Reply:
This function was refactored here: #504 Specifically here: 30da8ff